fileio/private/xsens_mvnx.m

```diff
@@ -0,0 +1,328 @@
+      dat(i:i+nchan-1,:) = reshape([mvnx.subject.frames.frame(idx).angularVelocity], [nchan, nsample]);
+      i=i+nchan;
+  end
+  
+  if isfield(mvnx.subject.frames.frame,'angularAcceleration')
+      nchan=length(find(contains(hdr.label, '_angularAcceleration')));
+      dat(i:i+nchan-1,:) = reshape([mvnx.subject.frames.frame(idx).angularAcceleration], [nchan, nsample]);
+      i=i+nchan;
+  end
+  
+  if isfield(mvnx.subject.frames.frame,'footContacts')
+      nchan=length(find(contains(hdr.label, '_footContacts')));
+      dat(i:i+nchan-1,:) = reshape([mvnx.subject.frames.frame(idx).footContacts], [nchan, nsample]);
+      i=i+nchan;
+  end
+  
+  if isfield(mvnx.subject.frames.frame,'sensorFreeAcceleration')
+      nchan=length(find(contains(hdr.label, '_sensorFreeAcceleration')));
+      dat(i:i+nchan-1,:) = reshape([mvnx.subject.frames.frame(idx).sensorFreeAcceleration], [nchan, nsample]);
+      i=i+nchan;
+  end
+  
+  if isfield(mvnx.subject.frames.frame,'sensorMagneticField')
+      nchan=length(find(contains(hdr.label, '_sensorMagneticField')));
+      dat(i:i+nchan-1,:) = reshape([mvnx.subject.frames.frame(idx).sensorMagneticField], [nchan, nsample]);
+      i=i+nchan;
+  end
+  
+  if isfield(mvnx.subject.frames.frame,'sensorOrientation')
+      nchan=length(find(contains(hdr.label, '_sensorOrientation')));
+      dat(i:i+nchan-1,:) = reshape([mvnx.subject.frames.frame(idx).sensorOrientation], [nchan, nsample]);
+      i=i+nchan;
+  end
+  
+  if isfield(mvnx.subject.frames.frame,'jointAngle')
+      nchan=length(find(contains(hdr.label, '_jointAngle_')));
+      dat(i:i+nchan-1,:) = reshape([mvnx.subject.frames.frame(idx).jointAngle], [nchan, nsample]);
+      i=i+nchan;
+  end
+  
+  if isfield(mvnx.subject.frames.frame,'jointAngleXZY')
+      nchan=length(find(contains(hdr.label, '_jointAngleXZY')));
+      dat(i:i+nchan-1,:) = reshape([mvnx.subject.frames.frame(idx).jointAngleXZY], [nchan, nsample]);
+      i=i+nchan;
+  end
+  
+  if isfield(mvnx.subject.frames.frame,'jointAngleErgo')
+      nchan=length(find(contains(hdr.label, '_jointAngleErgo_')));
+      dat(i:i+nchan-1,:) = reshape([mvnx.subject.frames.frame(idx).jointAngleErgo], [nchan, nsample]);
+      i=i+nchan;
+  end
+  
+  if isfield(mvnx.subject.frames.frame,'jointAngleErgoXZY')
+      nchan=length(find(contains(hdr.label, '_jointAngleErgoXZY')));
+      dat(i:i+nchan-1,:) = reshape([mvnx.subject.frames.frame(idx).jointAngleErgoXZY], [nchan, nsample]);
+      i=i+nchan;
+  end
+  
+  if isfield(mvnx.subject.frames.frame,'centerOfMass')
+      nchan=length(find(contains(hdr.label, '_centerOfMass')));
+      dat(i:i+nchan-1,:) = reshape([mvnx.subject.frames.frame(idx).centerOfMass], [nchan, nsample]);
+      i=i+nchan;
+  end
+  
+  % only select data that is asked for
+  dat = dat(chanindx, begsample:endsample);
+  
+  % return the data
+  varargout = {dat};
+  
+elseif needevt
+  %% parse the events
+  ft_warning('reading of events is not yet implemented');
```

Comment:
hier zouden misschien de stap-events en zo kunnen komen. Maar dat kan later nog wel.

Reply:
Ja, zeker! Maar eerst even nagaan of dit zou werken zo.

Reply:
Ik wilde trouwens een voorbeeld bijvoegen, maar het bestand is te groot...

Reply:
ziet er goed uit hoor!